supplementary_style_guide/glossary_terms_conventions/general_conventions/p.adoc

```diff
@@ -31,6 +31,16 @@
 
 *See also*:
 
+[discrete]
+[[persistent-storage]]
+==== image:images/yes.png[yes] Persistent storage (noun)
+*Description*: A type of storage that persists data even after the computer is turned off or after the process that created them ended, for example hard disk, tape or flash memory.
```

Comment:
```suggestion
*Description*: A type of storage that persists data even after the computer is turned off or after the process that created the data ended, for example hard disk, tape, or flash memory.
```

Reply:
s/them/the data because it wasn't clear what the pronoun was referencing. Also I'm a bit of a stickler for the Oxford comma.